samples/bluetooth/fast_pair/locator_tag/VERSION

```diff
@@ -1,5 +1,5 @@
 VERSION_MAJOR = 2
-VERSION_MINOR = 7
-PATCHLEVEL = 99
+VERSION_MINOR = 8
```

Comment:
Align also the Kconfigs in the following files:

```nrf/samples/bluetooth/fast_pair/locator_tag/configuration/prj.conf```
```nrf/samples/bluetooth/fast_pair/locator_tag/configuration/prj_release.conf```

```
CONFIG_BT_FAST_PAIR_FMDN_DULT_FIRMWARE_VERSION_MAJOR=2
CONFIG_BT_FAST_PAIR_FMDN_DULT_FIRMWARE_VERSION_MINOR=7
CONFIG_BT_FAST_PAIR_FMDN_DULT_FIRMWARE_VERSION_REVISION=99
```

->

```
CONFIG_BT_FAST_PAIR_FMDN_DULT_FIRMWARE_VERSION_MAJOR=2
CONFIG_BT_FAST_PAIR_FMDN_DULT_FIRMWARE_VERSION_MINOR=8
CONFIG_BT_FAST_PAIR_FMDN_DULT_FIRMWARE_VERSION_REVISION=0
```

Reply:
Updated